### PR TITLE
doc updated

### PR DESCRIPTION
**Pull request by ankitdas13**

### CODE REVIEW

File: documents/card.md [hunk 1]
```diff
@@ -371,7 +372,7 @@ Invoice invoice = instance.invoices.notifyBy(invoiceId,medium);
 ```java
 String invoiceId = "inv_FHrXGIpd3N17DX";
 
-instance.invoices.cancel(invoiceId)
+Invoice invoice = instance.invoices.cancel(invoiceId)
```
Comment (ersinghlovepreet) [inline documents/card.md:375]: this cancel is also delete or why it is changed?

File: documents/card.md [hunk 2]
```diff
@@ -591,7 +592,7 @@ String customerId = "cust_Hwq7Ba6TDXl1ga";
 
 String tokenId = "token_1Aa00000000001";
 
-Customer customer = instance.customers.deleteToken(customerId,tokenId)
+JSONObject customer = instance.customers.deleteToken(customerId,tokenId)
```
Comment (ersinghlovepreet) [inline documents/card.md:595]: why JSONObject?

File: documents/papernach.md [hunk 3]
```diff
@@ -594,7 +605,7 @@ String customerId = "cust_DtHaBuooGHTuyZ";
 
 String tokenId = "token_EhYgIE3pOyMQpD";
 
-instance.customers.deleteToken(customerId, tokenId);
+JSONObject customer = instance.customers.deleteToken(customerId, tokenId);
```
Comment (ersinghlovepreet) [inline documents/papernach.md:608]: JsonObject?

File: documents/registerEmandate.md [hunk 4]
```diff
@@ -433,7 +435,7 @@ String customerId = "cust_BMB3EwbqnqZ2EI";
 
 String tokenId = "token_FHf94Uym9tdYFJ";
 
-instance.Customers.deleteToken(customerId, tokenId);
+JSONObject customer = instance.customers.deleteToken(customerId, tokenId);
```
Comment (ersinghlovepreet) [inline documents/registerEmandate.md:438]: JsonObject?

File: documents/registerNach.md [hunk 5]
```diff
@@ -644,7 +655,7 @@ String customerId = "cust_BMB3EwbqnqZ2EI";
 
 String tokenId = "token_FHf94Uym9tdYFJ";
 
-instance.customers.deleteToken(customerId, tokenId);
+JSONObject customer = instance.customers.deleteToken(customerId, tokenId);
```
Comment (ersinghlovepreet) [inline documents/registerNach.md:658]: JsonOBject?

File: documents/card.md (hunk 1 above)
Comment (ankitdas13) [reply documents/card.md:375]: this is not delete. i forget to add variable there

File: documents/virtualAccount.md [hunk 6]
```diff
@@ -507,7 +508,7 @@ For add receiver to an existing virtual account response please click [here](htt
 String virtualId = "va_Di5gbNptcWV8fQ";
 
 String jsonRequest = "{\n" +
-            "  \"types\": \"bank_account\",\n" +
```
[the remark this answers is not in the document]
Comment (ankitdas13) [reply documents/virtualAccount.md:510]: its `add allowed payer` api .could you please check this [doc](https://razorpay.com/docs/smart-collect/api/add-tpv/#request-parameters)

File: documents/card.md (hunk 2 above)
Comment (ankitdas13) [reply documents/card.md]: removed JSONObject

File: documents/emandate.md [hunk 7]
```diff
@@ -372,7 +374,7 @@ String customerId = "cust_DtHaBuooGHTuyZ";
 
 String tokenId = "token_FHf94Uym9tdYFJ";
 
-Customer customer = instance.customers.deleteToken(customerId, tokenId);
```
[the remark this answers is not in the document]
Comment (ankitdas13) [reply documents/emandate.md]: removed JSONObject

File: documents/papernach.md (hunk 3 above)
Comment (ankitdas13) [reply documents/papernach.md]: removed JSONObject

File: documents/registerEmandate.md (hunk 4 above)
Comment (ankitdas13) [reply documents/registerEmandate.md]: made correction. JSONObject to Customer

File: documents/registerNach.md (hunk 5 above)
Comment (ankitdas13) [reply documents/registerNach.md]: removed JSONObject

**Pull request merged**